src/network/connection.cpp

```diff
@@ -710,6 +728,32 @@ void Connection::update() {
 			}
 
 			break;
+		}
+
+		case PACKET_SET_CONFIG_FLAG: {
+			// Packet type (4) + Packet number (8) + sensor_id(1) + flag_id (2) + state (1)
+			if (len < 16) {
+				m_Logger.warn("Invalid sensor config flag packet: too short");
+				break;
+			}
```

Comment:
Not ignoring bigger packets?

Reply:
we dont do that in any other packet, i think it makes sense to have this but shouldnt we make an issue for this so all packets have the same expected behavior on that regard?

Reply:
made an issue for this #342, will resolve this for now